docs/source/open-edx-background.rst

```diff
@@ -0,0 +1,39 @@
+Open edX Background
+===================
+
+What is Open edX?
+--------------------
+Open edX refers to its two components: the Open edX Platform and the Open edX Community.
+The Open edX Platform, a scaleable web-based platform that empowers both course authors and learners, consists of the Learning Management System (LMS) and the Content Management System (CMS).
+This open source project is supported by an international community working together to create this multiuse platform.
+The Open edX Community consists of tCRIL, the nonprofit organization that runs the project, and a number of core and individual contributors.
+For more information, see:
+
+- `What is "Open edX"? <https://open.edx.org/video/what-is-open-edx/>`_
+- `What makes the Open edX Platform unique? <https://open.edx.org/blog/what-makes-the-open-edx-platform-unique/>`_
+- `State of Open edX Community and Architecture <https://www.youtube.com/watch?v=HPrMYdwLsdE>`_
+
+More About the Open edX Community
+---------------------------------
+The Open edX community is international and includes those writing code to
+further the Open edX platform's capabilities, those authoring courses on a wide
+variety of Open edX instances, operators of those instances, instructional
+designers, translators, and more. The community is furthering the Open edX
+mission and expanding the reach and impact via numerous `working groups
+<https://openedx.atlassian.net/wiki/spaces/COMM/pages/46793351/Working+groups>`_
+and through the growing `core contributor program
+<https://openedx.atlassian.net/wiki/spaces/COMM/pages/3143205354/Open+edX+Core+Contributor+Program>`_.
+
+More About the Technical Side of the Open edX Platform
+------------------------------------------------------
+The Open edX Platform has a reasonably large codebase consisting of a monolith and a variety of microservices.
+The main platform code is hosted on GitHub `here <#>`_.
```

Comment:
What link are you using here?

Reply:
Woops, forgot to add that link. I was thinking https://github.com/edx/edx-platform

Reply:
We're moving the repo within the month - should we just use the new URL here, or leave the old and update it later?

Reply:
Probably just use the new one. In the case that we onboard someone unfamiliar with moving repos from edx to openedx before they are moved, we can always explain it quickly.

Reply:
link added